va/va_test.go

```diff
@@ -113,6 +116,11 @@ func setup(srv *httptest.Server, userAgent string, remoteVAs []RemoteVA, mockDNS
 		userAgent = "user agent 1.0"
 	}
 
+	perspective := PrimaryPerspective
+	if len(remoteVAs) == 0 {
+		perspective = "example perspective"
```

Comment:
Shouldn't this be a semi-random string so that we can simulate the various remotes having _different_ perspectives? Otherwise tests should fail when they attempt to verify that we got results from enough distinct perspectives.

Reply:
Good call. As of right now there aren't tests that check for that, but there will be soon.